osu.Game/Overlays/ModSelection/ModButton.cs

```diff
@@ -0,0 +1,80 @@
+﻿//Copyright (c) 2007-2016 ppy Pty Ltd <contact@ppy.sh>.
+//Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using OpenTK.Graphics;
+using osu.Framework;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Primitives;
+using osu.Framework.Input;
+using osu.Game.Modes;
+using osu.Game.Modes.UI;
+
+namespace osu.Game.Overlays.ModSelection
+{
+    class ModButton : ClickableContainer, IStateful<ModButtonState>
+    {
+        private Mod Mod;
+        private Color4 bgColor;
+
+        public ModButtonState State{get; set;}
+
+        private const int transform_time = 150;
+
+        public ModButton(Mod mod, Color4 color, bool isActive)
+        {
+            Mod = mod;
+            bgColor = color;
+
+            State = ModButtonState.Disabled;
+
+            RelativeSizeAxes = Axes.Y;

```

Comment:
why not just specify a Y as well so they don't push other buttons around when expanding?